src/loaders/common/loader_common_names.py

```diff
@@ -83,4 +83,18 @@
 MASH_METADATA = 'mash_run_metadata.json'
 
 # The fatal error file created if a data file cannot be successfully processed
-FATAL_ERROR_FILE = "fatal_error.json"
\ No newline at end of file
+FATAL_ERROR_FILE = "fatal_error.json"
+
+# key name for sample file and prepared sample file in the metadata file for downloaded workspace objects
+SAMPLE_FILE_KEY = "sample_file"
+SAMPLE_PREPARED_KEY = "sample_prepared_file"
+
+# extension for source sample data file and prepared sample node data file for downloaded workspace objects
+SAMPLE_FILE_EXT = "sample"
+SAMPLE_PREPARED_EXT = "prepared.sample"
+
+# key name for latitude and longitude data from sample's meta_controlled
+SAMPLE_LATITUDE = "latitude"
+SAMPLE_LONGITUDE = "longitude"
+# key name for created sample geo-spatial data in format of [longitude, latitude]
+SAMPLE_GEO = 'geo_spatial'
```

Comment:
```suggestion
SAMPLE_GEO = '_geo_spatial'
```

To indicate that this field shouldn't be returned from the API

Reply:
👍

Reply:
:+1: